### ISSUE TITLE
Cannot deserialize value of type `com.github.dockerjava.api.model.Capability`

### ISSUE THREAD

**Issue by royalahn** [Krillsson/sys-API#144]
Hello developer, 

I like your app Monitee that it's useful to monitoring my servers, thank you to develop this app.

I've updated Monitee `0.12.1` and found that not supported old versions.

<ins>**So, I updated docker-compose and config files for sprintboot but some errors occurs below.**</ins>

Please check it and let me know how to solve it.

Thank you in advance.

<details><summary>Logs</summary>

```java
  ______ ______ ______ ______ ______ ______ ______
 |______|______|______|______|______|______|______|
 | |                                            | |
 | |    / ____|         | |                     | |
 | |   | (___  _   _ ___| |_ ___ _ __ ___       | |
 | |    \___ \| | | / __| __/ _ | '_ ` _ \      | |
 | |    ____) | |_| \__ | ||  __| | | | | |     | |
 | |   |_____/ \__, |___/\__\___|_| |_| |_|     | |
 | |            __/ |                           | |
 |_|           |___/__ _____                    |_|
 | |       /\   |  __ |_   _|                   | |
 | |      /  \  | |__) || |                     | |
 | |     / /\ \ |  ___/ | |                     | |
 | |    / ____ \| |    _| |_                    | |
 | |   /_/    \_|_|   |_____|                   | |
 | |                                            | |
 |_|____ ______ ______ ______ ______ ______ ____|_|
 |______|______|______|______|______|______|______|


2024-06-15T05:44:09.158Z  INFO 1325169 --- [sysapi] [           main] c.krillsson.sysapi.SysAPIApplicationKt   : Starting SysAPIApplicationKt using Java 21.0.2 with PID 1325169 (/app/classes started by root in /)
2024-06-15T05:44:09.166Z  INFO 1325169 --- [sysapi] [           main] c.krillsson.sysapi.SysAPIApplicationKt   : No active profile set, falling back to 1 default profile: "default"
2024-06-15T05:44:11.340Z  INFO 1325169 --- [sysapi] [           main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2024-06-15T05:44:11.578Z  INFO 1325169 --- [sysapi] [           main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 230 ms. Found 8 JPA repository interfaces.
2024-06-15T05:44:14.212Z  INFO 1325169 --- [sysapi] [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with ports 8444 (https), 8088 (http)
2024-06-15T05:44:14.234Z  INFO 1325169 --- [sysapi] [           main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2024-06-15T05:44:14.235Z  INFO 1325169 --- [sysapi] [           main] o.apache.catalina.core.StandardEngine    : Starting Servlet engine: [Apache Tomcat/10.1.19]
2024-06-15T05:44:14.315Z  INFO 1325169 --- [sysapi] [           main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2024-06-15T05:44:14.315Z  INFO 1325169 --- [sysapi] [           main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 5012 ms
2024-06-15T05:44:14.588Z  INFO 1325169 --- [sysapi] [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Starting...
2024-06-15T05:44:14.798Z  INFO 1325169 --- [sysapi] [           main] com.zaxxer.hikari.pool.HikariPool        : HikariPool-1 - Added connection org.sqlite.jdbc4.JDBC4Connection@40b54762
2024-06-15T05:44:14.802Z  INFO 1325169 --- [sysapi] [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Start completed.
2024-06-15T05:44:14.830Z  INFO 1325169 --- [sysapi] [           main] org.flywaydb.core.FlywayExecutor         : Database: jdbc:sqlite:data/database.sqlite (SQLite 3.45)
2024-06-15T05:44:14.885Z  INFO 1325169 --- [sysapi] [           main] o.f.core.internal.command.DbValidate     : Successfully validated 10 migrations (execution time 00:00.034s)
2024-06-15T05:44:14.893Z  INFO 1325169 --- [sysapi] [           main] o.f.core.internal.command.DbMigrate      : Current version of schema "main": 10
2024-06-15T05:44:14.894Z  INFO 1325169 --- [sysapi] [           main] o.f.core.internal.command.DbMigrate      : Schema "main" is up to date. No migration necessary.
2024-06-15T05:44:15.044Z  INFO 1325169 --- [sysapi] [           main] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]
2024-06-15T05:44:15.100Z  INFO 1325169 --- [sysapi] [           main] org.hibernate.Version                    : HHH000412: Hibernate ORM core version 6.4.2.Final
2024-06-15T05:44:15.135Z  INFO 1325169 --- [sysapi] [           main] o.h.c.internal.RegionFactoryInitiator    : HHH000026: Second-level cache disabled
2024-06-15T05:44:15.454Z  INFO 1325169 --- [sysapi] [           main] o.s.o.j.p.SpringPersistenceUnitInfo      : No LoadTimeWeaver setup: ignoring JPA class transformer
2024-06-15T05:44:17.593Z  INFO 1325169 --- [sysapi] [           main] o.h.e.t.j.p.i.JtaPlatformInitiator       : HHH000489: No JTA platform available (set 'hibernate.transaction.jta.platform' to enable JTA platform integration)
2024-06-15T05:44:17.858Z  INFO 1325169 --- [sysapi] [           main] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
/usr/bin/systemctl
2024-06-15T05:44:27.363Z  INFO 1325169 --- [sysapi] [         task-1] com.krillsson.sysapi.mdns.Mdns           : Registered mDNS: _https._tcp.local with name: royal-arm64-u2 - sys-API https at port 8444 (took 7584ms)
2024-06-15T05:44:28.964Z  INFO 1325169 --- [sysapi] [           main] com.krillsson.sysapi.upnp.UpnpIgd        : No active gateway device found
2024-06-15T05:44:28.992Z  WARN 1325169 --- [sysapi] [           main] JpaBaseConfiguration$JpaWebConfiguration : spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
2024-06-15T05:44:29.670Z  INFO 1325169 --- [sysapi] [           main] efaultSchemaResourceGraphQlSourceBuilder : Loaded 7 resource(s) in the GraphQL schema.
2024-06-15T05:44:29.939Z  INFO 1325169 --- [sysapi] [         task-1] com.krillsson.sysapi.mdns.Mdns           : Registered mDNS: _http._tcp.local with name: royal-arm64-u2 - sys-API http at port 8088 (took 2574ms)
2024-06-15T05:44:30.683Z  INFO 1325169 --- [sysapi] [           main] o.s.b.a.g.GraphQlAutoConfiguration       : GraphQL schema inspection:
	Unmapped fields: {}
	Unmapped registrations: {}
	Unmapped arguments: {}
	Skipped types: []
2024-06-15T05:44:30.718Z  INFO 1325169 --- [sysapi] [           main] s.b.a.g.s.GraphQlWebMvcAutoConfiguration : GraphQL endpoint HTTP POST /graphql
2024-06-15T05:44:30.779Z  INFO 1325169 --- [sysapi] [           main] o.s.s.web.DefaultSecurityFilterChain     : Will secure any request with [org.springframework.security.web.session.DisableEncodeUrlFilter@1231e446, org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter@32c60fec, org.springframework.security.web.context.SecurityContextHolderFilter@32561cd5, org.springframework.security.web.header.HeaderWriterFilter@183a593f, org.springframework.web.filter.CorsFilter@4d5a21cc, org.springframework.security.web.authentication.www.BasicAuthenticationFilter@9252e52, org.springframework.security.web.authentication.AnonymousAuthenticationFilter@3990c36c, org.springframework.security.web.access.ExceptionTranslationFilter@60a0094a, org.springframework.security.web.access.intercept.AuthorizationFilter@4a218cc6]
2024-06-15T05:44:31.254Z  INFO 1325169 --- [sysapi] [           main] o.a.t.util.net.NioEndpoint.certificate   : Connector [https-jsse-nio-8444], TLS virtual host [_default_], certificate type [UNDEFINED] configured from keystore [/root/.keystore] using alias [sys-api-key] with trust store [null]
2024-06-15T05:44:31.272Z  INFO 1325169 --- [sysapi] [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on ports 8444 (https), 8088 (http) with context path '/'
2024-06-15T05:44:31.314Z  INFO 1325169 --- [sysapi] [           main] c.krillsson.sysapi.SysAPIApplicationKt   : Started SysAPIApplicationKt in 22.723 seconds (process running for 23.212)
2024-06-15T05:44:32.270Z ERROR 1325169 --- [sysapi] [   scheduling-2] o.s.s.s.TaskUtils$LoggingErrorHandler    : Unexpected error occurred in scheduled task

java.lang.RuntimeException: com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type `com.github.dockerjava.api.model.Capability` from String "all": not one of the values accepted for Enum class: [ALL, SYS_BOOT, DAC_OVERRIDE, NET_RAW, BLOCK_SUSPEND, FOWNER, IPC_LOCK, IPC_OWNER, SYS_PACCT, NET_BIND_SERVICE, WAKE_ALARM, FSETID, DAC_READ_SEARCH, SYS_CHROOT, AUDIT_READ, SYS_RAWIO, SYS_ADMIN, KILL, MAC_ADMIN, SYS_RESOURCE, CHOWN, PERFMON, SETPCAP, SYS_PTRACE, NET_ADMIN, SETFCAP, SYS_NICE, LINUX_IMMUTABLE, BPF, AUDIT_CONTROL, LEASE, AUDIT_WRITE, SYS_MODULE, MKNOD, SYSLOG, MAC_OVERRIDE, SYS_TIME, SETGID, SETUID, CHECKPOINT_RESTORE, SYS_TTY_CONFIG, NET_BROADCAST]
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 2072] (through reference chain: com.github.dockerjava.api.command.InspectContainerResponse["HostConfig"]->com.github.dockerjava.api.model.HostConfig["CapDrop"]->java.lang.Object[][0])
	at com.github.dockerjava.core.DefaultInvocationBuilder.get(DefaultInvocationBuilder.java:77) ~[docker-java-core-3.3.6.jar:na]
	at com.github.dockerjava.core.exec.InspectContainerCmdExec.execute(InspectContainerCmdExec.java:31) ~[docker-java-core-3.3.6.jar:na]
	at com.github.dockerjava.core.exec.InspectContainerCmdExec.execute(InspectContainerCmdExec.java:13) ~[docker-java-core-3.3.6.jar:na]
	at com.github.dockerjava.core.exec.AbstrSyncDockerCmdExec.exec(AbstrSyncDockerCmdExec.java:21) ~[docker-java-core-3.3.6.jar:na]
	at com.github.dockerjava.core.command.AbstrDockerCmd.exec(AbstrDockerCmd.java:33) ~[docker-java-core-3.3.6.jar:na]
	at com.github.dockerjava.core.command.InspectContainerCmdImpl.exec(InspectContainerCmdImpl.java:51) ~[docker-java-core-3.3.6.jar:na]
	at com.krillsson.sysapi.docker.DockerClient.listContainers(DockerClient.kt:102) ~[classes/:na]
	at com.krillsson.sysapi.docker.DockerClient.listContainers$default(DockerClient.kt:91) ~[classes/:na]
	at com.krillsson.sysapi.docker.ContainerManager.containersCache$lambda$0(ContainerManager.kt:36) ~[classes/:na]
	at com.google.common.base.Suppliers$ExpiringMemoizingSupplier.get(Suppliers.java:199) ~[guava-19.0.jar:na]
	at com.krillsson.sysapi.docker.ContainerManager.containers(ContainerManager.kt:61) ~[classes/:na]
	at com.krillsson.sysapi.docker.ContainerStatisticsHistoryRecorder.run(ContainerStatisticsHistoryRecorder.kt:21) ~[classes/:na]
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103) ~[na:na]
	at java.base/java.lang.reflect.Method.invoke(Method.java:580) ~[na:na]
	at org.springframework.scheduling.support.ScheduledMethodRunnable.runInternal(ScheduledMethodRunnable.java:130) ~[spring-context-6.1.5.jar:6.1.5]
	at org.springframework.scheduling.support.ScheduledMethodRunnable.lambda$run$2(ScheduledMethodRunnable.java:124) ~[spring-context-6.1.5.jar:6.1.5]
	at io.micrometer.observation.Observation.observe(Observation.java:499) ~[micrometer-observation-1.13.0-M2.jar:1.13.0-M2]
	at org.springframework.scheduling.support.ScheduledMethodRunnable.run(ScheduledMethodRunnable.java:124) ~[spring-context-6.1.5.jar:6.1.5]
	at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54) ~[spring-context-6.1.5.jar:6.1.5]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572) ~[na:na]
	at java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:358) ~[na:na]
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642) ~[na:na]
	at java.base/java.lang.Thread.run(Thread.java:1583) ~[na:na]
Caused by: com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type `com.github.dockerjava.api.model.Capability` from String "all": not one of the values accepted for Enum class: [ALL, SYS_BOOT, DAC_OVERRIDE, NET_RAW, BLOCK_SUSPEND, FOWNER, IPC_LOCK, IPC_OWNER, SYS_PACCT, NET_BIND_SERVICE, WAKE_ALARM, FSETID, DAC_READ_SEARCH, SYS_CHROOT, AUDIT_READ, SYS_RAWIO, SYS_ADMIN, KILL, MAC_ADMIN, SYS_RESOURCE, CHOWN, PERFMON, SETPCAP, SYS_PTRACE, NET_ADMIN, SETFCAP, SYS_NICE, LINUX_IMMUTABLE, BPF, AUDIT_CONTROL, LEASE, AUDIT_WRITE, SYS_MODULE, MKNOD, SYSLOG, MAC_OVERRIDE, SYS_TIME, SETGID, SETUID, CHECKPOINT_RESTORE, SYS_TTY_CONFIG, NET_BROADCAST]
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 2072] (through reference chain: com.github.dockerjava.api.command.InspectContainerResponse["HostConfig"]->com.github.dockerjava.api.model.HostConfig["CapDrop"]->java.lang.Object[][0])
	at com.fasterxml.jackson.databind.exc.InvalidFormatException.from(InvalidFormatException.java:67) ~[jackson-databind-2.17.0.jar:2.17.0]
	at com.fasterxml.jackson.databind.DeserializationContext.weirdStringException(DeserializationContext.java:1958) ~[jackson-databind-2.17.0.jar:2.17.0]
	at com.fasterxml.jackson.databind.DeserializationContext.handleWeirdStringValue(DeserializationContext.java:1245) ~[jackson-databind-2.17.0.jar:2.17.0]
	at com.fasterxml.jackson.databind.deser.std.EnumDeserializer._deserializeAltString(EnumDeserializer.java:447) ~[jackson-databind-2.17.0.jar:2.17.0]
	at com.fasterxml.jackson.databind.deser.std.EnumDeserializer._fromString(EnumDeserializer.java:304) ~[jackson-databind-2.17.0.jar:2.17.0]
	at com.fasterxml.jackson.databind.deser.std.EnumDeserializer.deserialize(EnumDeserializer.java:273) ~[jackson-databind-2.17.0.jar:2.17.0]
	at com.fasterxml.jackson.databind.deser.std.ObjectArrayDeserializer.deserialize(ObjectArrayDeserializer.java:218) ~[jackson-databind-2.17.0.jar:2.17.0]
	at com.fasterxml.jackson.databind.deser.impl.FieldProperty.deserializeAndSet(FieldProperty.java:138) ~[jackson-databind-2.17.0.jar:2.17.0]
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserializeFromObject(BeanDeserializer.java:393) ~[jackson-databind-2.17.0.jar:2.17.0]
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:185) ~[jackson-databind-2.17.0.jar:2.17.0]
	at com.fasterxml.jackson.databind.deser.impl.FieldProperty.deserializeAndSet(FieldProperty.java:138) ~[jackson-databind-2.17.0.jar:2.17.0]
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserializeFromObject(BeanDeserializer.java:393) ~[jackson-databind-2.17.0.jar:2.17.0]
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:185) ~[jackson-databind-2.17.0.jar:2.17.0]
	at com.fasterxml.jackson.databind.deser.DefaultDeserializationContext.readRootValue(DefaultDeserializationContext.java:342) ~[jackson-databind-2.17.0.jar:2.17.0]
	at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4905) ~[jackson-databind-2.17.0.jar:2.17.0]
	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3893) ~[jackson-databind-2.17.0.jar:2.17.0]
	at com.github.dockerjava.core.DefaultInvocationBuilder.get(DefaultInvocationBuilder.java:75) ~[docker-java-core-3.3.6.jar:na]
	... 24 common frames omitted

```

</details>

**Comment by Krillsson** [comment]
Can you add these lines at the bottom of your application.properties file inside /config directory?

```
# docker java client fixes
spring.jackson.mapper.ACCEPT_CASE_INSENSITIVE_ENUMS=true
spring.jackson.mapper.ACCEPT_CASE_INSENSITIVE_PROPERTIES=true
spring.jackson.mapper.ACCEPT_CASE_INSENSITIVE_VALUES=true
```



![Image](https://github.com/Krillsson/sys-API/assets/2466419/f7fb08bd-1b28-4b01-a80c-980b3e52af97)

**Comment by royalahn** [comment]
> Can you add these lines at the bottom of your application.properties file inside /config directory?
> 
> ```
> # docker java client fixes
> spring.jackson.mapper.ACCEPT_CASE_INSENSITIVE_ENUMS=true
> spring.jackson.mapper.ACCEPT_CASE_INSENSITIVE_PROPERTIES=true
> spring.jackson.mapper.ACCEPT_CASE_INSENSITIVE_VALUES=true
> ```
> 
> ![Image](https://private-user-images.githubusercontent.com/2466419/340355795-f7fb08bd-1b28-4b01-a80c-980b3e52af97.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg2NzgwNjEsIm5iZiI6MTcxODY3Nzc2MSwicGF0aCI6Ii8yNDY2NDE5LzM0MDM1NTc5NS1mN2ZiMDhiZC0xYjI4LTRiMDEtYTgwYy05ODBiM2U1MmFmOTcucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDYxOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA2MThUMDIyOTIxWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MDY2ZjYwN2Q0YmY1ZTJkN2Q3MzM0NWRiY2NmODRhNGFmNzYzNTU2OWEwNjgzMzRjYzU4YzA3OTQxOWI1YzEzNSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.fYyhGEeCucHaXIwrBIWgi_7vXw4F3KACkRh4IBbwDb8)

Okay, I will try it and let you know soon :)

**Comment by royalahn** [comment]
> Can you add these lines at the bottom of your application.properties file inside /config directory?
> 
> ```
> # docker java client fixes
> spring.jackson.mapper.ACCEPT_CASE_INSENSITIVE_ENUMS=true
> spring.jackson.mapper.ACCEPT_CASE_INSENSITIVE_PROPERTIES=true
> spring.jackson.mapper.ACCEPT_CASE_INSENSITIVE_VALUES=true
> ```
> 

It works well.

<details><summary>Logs</summary>

```java
  ______ ______ ______ ______ ______ ______ ______
 |______|______|______|______|______|______|______|
 | |                                            | |
 | |    / ____|         | |                     | |
 | |   | (___  _   _ ___| |_ ___ _ __ ___       | |
 | |    \___ \| | | / __| __/ _ | '_ ` _ \      | |
 | |    ____) | |_| \__ | ||  __| | | | | |     | |
 | |   |_____/ \__, |___/\__\___|_| |_| |_|     | |
 | |            __/ |                           | |
 |_|           |___/__ _____                    |_|
 | |       /\   |  __ |_   _|                   | |
 | |      /  \  | |__) || |                     | |
 | |     / /\ \ |  ___/ | |                     | |
 | |    / ____ \| |    _| |_                    | |
 | |   /_/    \_|_|   |_____|                   | |
 | |                                            | |
 |_|____ ______ ______ ______ ______ ______ ____|_|
 |______|______|______|______|______|______|______|


2024-06-18T13:45:32.579Z  INFO 1828939 --- [sysapi] [           main] c.krillsson.sysapi.SysAPIApplicationKt   : Starting SysAPIApplicationKt using Java 21.0.2 with PID 1828939 (/app/classes started by root in /)
2024-06-18T13:45:32.599Z  INFO 1828939 --- [sysapi] [           main] c.krillsson.sysapi.SysAPIApplicationKt   : No active profile set, falling back to 1 default profile: "default"
2024-06-18T13:45:34.810Z  INFO 1828939 --- [sysapi] [           main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2024-06-18T13:45:35.066Z  INFO 1828939 --- [sysapi] [           main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 246 ms. Found 8 JPA repository interfaces.
2024-06-18T13:45:37.576Z  INFO 1828939 --- [sysapi] [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with ports 8444 (https), 8088 (http)
2024-06-18T13:45:37.596Z  INFO 1828939 --- [sysapi] [           main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2024-06-18T13:45:37.597Z  INFO 1828939 --- [sysapi] [           main] o.apache.catalina.core.StandardEngine    : Starting Servlet engine: [Apache Tomcat/10.1.19]
2024-06-18T13:45:37.678Z  INFO 1828939 --- [sysapi] [           main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2024-06-18T13:45:37.678Z  INFO 1828939 --- [sysapi] [           main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 4936 ms
2024-06-18T13:45:37.948Z  INFO 1828939 --- [sysapi] [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Starting...
2024-06-18T13:45:38.075Z  INFO 1828939 --- [sysapi] [           main] com.zaxxer.hikari.pool.HikariPool        : HikariPool-1 - Added connection org.sqlite.jdbc4.JDBC4Connection@181098bf
2024-06-18T13:45:38.077Z  INFO 1828939 --- [sysapi] [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Start completed.
2024-06-18T13:45:38.105Z  INFO 1828939 --- [sysapi] [           main] org.flywaydb.core.FlywayExecutor         : Database: jdbc:sqlite:data/database.sqlite (SQLite 3.45)
2024-06-18T13:45:38.181Z  INFO 1828939 --- [sysapi] [           main] o.f.core.internal.command.DbValidate     : Successfully validated 10 migrations (execution time 00:00.055s)
2024-06-18T13:45:38.190Z  INFO 1828939 --- [sysapi] [           main] o.f.core.internal.command.DbMigrate      : Current version of schema "main": 10
2024-06-18T13:45:38.192Z  INFO 1828939 --- [sysapi] [           main] o.f.core.internal.command.DbMigrate      : Schema "main" is up to date. No migration necessary.
2024-06-18T13:45:38.336Z  INFO 1828939 --- [sysapi] [           main] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]
2024-06-18T13:45:38.414Z  INFO 1828939 --- [sysapi] [           main] org.hibernate.Version                    : HHH000412: Hibernate ORM core version 6.4.2.Final
2024-06-18T13:45:38.446Z  INFO 1828939 --- [sysapi] [           main] o.h.c.internal.RegionFactoryInitiator    : HHH000026: Second-level cache disabled
2024-06-18T13:45:38.667Z  INFO 1828939 --- [sysapi] [           main] o.s.o.j.p.SpringPersistenceUnitInfo      : No LoadTimeWeaver setup: ignoring JPA class transformer
2024-06-18T13:45:40.688Z  INFO 1828939 --- [sysapi] [           main] o.h.e.t.j.p.i.JtaPlatformInitiator       : HHH000489: No JTA platform available (set 'hibernate.transaction.jta.platform' to enable JTA platform integration)
2024-06-18T13:45:40.995Z  INFO 1828939 --- [sysapi] [           main] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
/usr/bin/journalctl
/usr/bin/systemctl
2024-06-18T13:45:50.646Z  INFO 1828939 --- [sysapi] [         task-1] com.krillsson.sysapi.mdns.Mdns           : Registered mDNS: _https._tcp.local with name: royal-arm64-u2 - sys-API https at port 8444 (took 7610ms)
2024-06-18T13:45:52.205Z  INFO 1828939 --- [sysapi] [           main] com.krillsson.sysapi.upnp.UpnpIgd        : No active gateway device found
2024-06-18T13:45:52.253Z  WARN 1828939 --- [sysapi] [           main] JpaBaseConfiguration$JpaWebConfiguration : spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
2024-06-18T13:45:52.997Z  INFO 1828939 --- [sysapi] [           main] efaultSchemaResourceGraphQlSourceBuilder : Loaded 7 resource(s) in the GraphQL schema.
2024-06-18T13:45:53.233Z  INFO 1828939 --- [sysapi] [         task-1] com.krillsson.sysapi.mdns.Mdns           : Registered mDNS: _http._tcp.local with name: royal-arm64-u2 - sys-API http at port 8088 (took 2586ms)
2024-06-18T13:45:54.165Z  INFO 1828939 --- [sysapi] [           main] o.s.b.a.g.GraphQlAutoConfiguration       : GraphQL schema inspection:
	Unmapped fields: {}
	Unmapped registrations: {}
	Unmapped arguments: {}
	Skipped types: []
2024-06-18T13:45:54.229Z  INFO 1828939 --- [sysapi] [           main] s.b.a.g.s.GraphQlWebMvcAutoConfiguration : GraphQL endpoint HTTP POST /graphql
2024-06-18T13:45:54.300Z  INFO 1828939 --- [sysapi] [           main] o.s.s.web.DefaultSecurityFilterChain     : Will secure any request with [org.springframework.security.web.session.DisableEncodeUrlFilter@604aa23e, org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter@414c2c8c, org.springframework.security.web.context.SecurityContextHolderFilter@2f931d5d, org.springframework.security.web.header.HeaderWriterFilter@63ad2a0f, org.springframework.web.filter.CorsFilter@73d9f235, org.springframework.security.web.authentication.www.BasicAuthenticationFilter@7b8b458e, org.springframework.security.web.authentication.AnonymousAuthenticationFilter@389ae8f1, org.springframework.security.web.access.ExceptionTranslationFilter@3810d15d, org.springframework.security.web.access.intercept.AuthorizationFilter@2625c1cc]
2024-06-18T13:45:54.781Z  INFO 1828939 --- [sysapi] [           main] o.a.t.util.net.NioEndpoint.certificate   : Connector [https-jsse-nio-8444], TLS virtual host [_default_], certificate type [UNDEFINED] configured from keystore [/root/.keystore] using alias [sys-api-key] with trust store [null]
2024-06-18T13:45:54.819Z  INFO 1828939 --- [sysapi] [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on ports 8444 (https), 8088 (http) with context path '/'
2024-06-18T13:45:54.915Z  INFO 1828939 --- [sysapi] [           main] c.krillsson.sysapi.SysAPIApplicationKt   : Started SysAPIApplicationKt in 22.929 seconds (process running for 23.502)
2024-06-18T13:45:56.963Z  INFO 1828939 --- [sysapi] [   scheduling-4] c.k.s.core.history.HistoryRepository     : Purging history older than 2024-06-04T13:45:56.963517510Z
2024-06-18T13:46:11.952Z  INFO 1828939 --- [sysapi] [nio-8088-exec-3] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring DispatcherServlet 'dispatcherServlet'
2024-06-18T13:46:11.960Z  INFO 1828939 --- [sysapi] [nio-8088-exec-3] o.s.web.servlet.DispatcherServlet        : Initializing Servlet 'dispatcherServlet'
2024-06-18T13:46:11.965Z  INFO 1828939 --- [sysapi] [nio-8088-exec-3] o.s.web.servlet.DispatcherServlet        : Completed initialization in 4 ms
2024-06-18T13:46:27.368Z  INFO 1828939 --- [sysapi] [nio-8088-exec-6] c.k.s.core.history.HistoryRepository     : Took 28ms to fetch 164 history entries
2024-06-18T13:46:27.445Z  INFO 1828939 --- [sysapi] [nio-8088-exec-6] c.k.s.core.history.HistoryRepository     : Took 6ms to fetch 24 history entries
2024-06-18T13:47:01.430Z  INFO 1828939 --- [sysapi] [   scheduling-2] c.k.s.c.h.ContainersHistoryRepository    : Purged 0 history older than 2024-06-04T13:47:01.428248861Z
```

</details>

Thank you very much ! 👍👍👍

![image](https://github.com/Krillsson/sys-API/assets/42069432/f184d62a-2c38-4e16-99fe-5f7f0093acbe)

**Comment by Krillsson** [comment]
Glad to hear it worked. Will leave the issue with the solution up until the next release

**Comment by Krillsson** [comment]
Released these changes in [0.31.1](https://github.com/Krillsson/sys-API/releases/tag/0.31.1) now
